src/config/convert.js

```diff
@@ -513,15 +513,16 @@ const vmess1Parser = (tag, params) => {
     "streamSettings": {
     }
   }
-  if (params.length > 1) {
-    return new Error('invalid vmess1 parameters')
-  }
-  const url = new URL(params[0].trim())
+  // workaround for HTTP/2 host handling, may contain comma
```

Comment:
since the vmess1 scheme is similar to URI, we should use percent-encoding for the parameters, i.e. encodeURIComponent('example.com,example1.com') --> 'example.com%2Cexample1.com'